server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+
+// TODO - Change this to use environment variables
+const port = 8081;
+
+const eventType = [ 
+    {label : "cop"},
+    {label : "landl"},
+    {label : "learningEvents"} 
+];
+
+// Dynamic endpoint receiving a param of eventtype to test
+app.get('/events/:eventtype', function (req, res) {
```

Comment:
`:eventtype` better write it in camel case `:eventType`, since it clashes with the array `eventType`, rename the latter as `eventTypes`

Reply:
Will do :)